src/commands/BugCommand.ts

```diff
@@ -30,39 +40,26 @@ export default class BugCommand extends PrefixCommand {
 			embed = await mention.getEmbed();
 		} catch ( err ) {
 			try {
-				await message.channel.send( err );
+				await interaction.reply( { content: err, ephemeral: true } );
 			} catch ( err ) {
 				Command.logger.log( err );
+				return false;
 			}
-			return false;
+			return true;
 		}
 
 		if ( embed === undefined ) return false;
 
-		embed.setFooter( { text: message.author.tag, iconURL: message.author.avatarURL() ?? undefined } )
-			.setTimestamp( message.createdAt );
+		embed.setFooter( { text: interaction.user.tag, iconURL: interaction.user.avatarURL() ?? undefined } )
+			.setTimestamp( interaction.createdAt );
 
 		try {
-			await message.channel.send( { embeds: [embed] } );
+			await interaction.reply( { embeds: [embed], ephemeral: true } );
```

Comment:
I think for this command it makes more sense to have the command reply public rather than ephermal.